projects/ad5754r_iio/README.txt

```diff
@@ -0,0 +1,41 @@
+Evaluation Boards/Products Supported
```

Comment:
Are we still following this structure for the readme?
It was suggested that we add a path to the project documentation in the readme

Reply:
> Are we still following this structure for the readme? It was suggested that we add a path to the project documentation in the readme

Do we need a readme file at all? I think for newer projects, we're not creating readme files!

Reply:
We decided to keep the README, but simply point to the actual documentation from there. https://github.com/analogdevicesinc/precision-converters-firmware/blob/main/projects/ad4170_iio/README.txt

Reply:
> We decided to keep the README, but simply point to the actual documentation from there. https://github.com/analogdevicesinc/precision-converters-firmware/blob/main/projects/ad4170_iio/README.txt

But now that we've links exposed in the repo about section and the workflows running, does it make sense to not have it at all?